frontend/src/BeltHelpers.res

```diff
@@ -12,3 +12,21 @@ module MapString = {
     Map.String.update(map, k, maybeAdd)
   }
 }
+
+exception StopIteration
+let arrayFindRev = (arr, pred) => {
```

Comment:
Would that make sense to rename this function to `arrayFindRevExn`? I guess there's probably no good "answer" and would mostly depend on conventions. But since I see that `Belt` tends to use this `Exn` suffix for functions that can raise exceptions, it would probably make things more consistent. What do you think?

Reply:
But does it really raise an exception? :)

This exception is internal and will under no circumstances be leaked to the user.

Reply:
In other words, this is how I normally implement `break` for for-loops given that it's not supported by OCaml.

Not sure if there's a better way.

Reply:
Oh yes indeed, sorry I'm not used to using exceptions for that. I would probably have written it like the following but don't know if that's better:

```res
let arrayFindRev = (arr, pred) => {
  let len = Belt.Array.length(arr)
  let rec loop = (i, out) =>
    switch i {
    | -1 => out
    | _ =>
      let item = Belt.Array.getExn(arr, i)
      pred(item) ? Some(item) : loop(i - 1, None)
    }
  loop(len - 1, None)
}
```

Reply:
I think you're version is better because it avoid the use of exception and the mutable ref. I'll submit commit to use it.